{
	"id": "AVG-2292",
	"advisories": [
		{
			"content": {
				"id": "AVG-2292",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "Critical"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-2292"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-37635",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37635"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37636",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37636"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37637",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37637"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37638",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37638"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37639",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37639"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37640",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37640"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37641",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37641"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37642",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37642"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37643",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37643"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37644",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37644"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37645",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37645"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37646",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37646"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37647",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37647"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37648",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37648"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37649",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37649"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37650",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37650"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37651",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37651"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37652",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37652"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37653",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37653"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37654",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37654"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37655",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37655"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37656",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37656"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37657",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37657"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37658",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37658"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37659",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37659"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37660",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37660"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37661",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37661"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37662",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37662"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37663",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37663"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37664",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37664"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37665",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37665"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37666",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37666"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37667",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37667"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37668",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37668"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37669",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37669"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37670",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37670"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37671",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37671"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37672",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37672"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37673",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37673"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37674",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37674"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37675",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37675"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37676",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37676"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37677",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37677"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37678",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37678"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37679",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37679"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37680",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37680"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37681",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37681"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37682",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37682"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37683",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37683"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37684",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37684"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37685",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37685"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37686",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37686"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37687",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37687"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37688",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37688"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37689",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37689"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37690",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37690"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37691",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37691"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		},
		{
			"content": {
				"id": "CVE-2021-37692",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2021-37692"
					}
				]
			},
			"segments": [
				{
					"ecosystem": "arch"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "arch",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterions": [
							{
								"vulnerable": true,
								"fix_status": {
									"class": "fixed"
								},
								"package": {
									"name": "tensorflow"
								},
								"affected": {
									"type": "pacman",
									"range": [
										{
											"lt": "2.5.1-1"
										}
									],
									"fixed": [
										"2.5.1-1"
									]
								}
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "arch",
		"raws": [
			"vuls-data-raw-arch/AVG-2292.json"
		]
	}
}
